### PR TITLE
Added navigation and a couple small style updates

### PR DESCRIPTION
**Pull request by merrimanr**
## Contributor Comments
This PR adds a simple horizontal nav to the top of the Alerts UI.  Should make getting to the PCAP query panel fairly intuitive.  I also cleaned up the styling a little bit to make it better match our existing style.

## Pull Request Checklist

Thank you for submitting a contribution to Apache Metron.  
Please refer to our [Development Guidelines](https://cwiki.apache.org/confluence/pages/viewpage.action?pageId=61332235) for the complete guide to follow for contributions.  
Please refer also to our [Build Verification Guidelines](https://cwiki.apache.org/confluence/display/METRON/Verifying+Builds?show-miniview) for complete smoke testing guides.  

In order to streamline the review of the contribution we ask you follow these guidelines and ask you to double check the following:

### For all changes:
- [ ] Is there a JIRA ticket associated with this PR? If not one needs to be created at [Metron Jira](https://issues.apache.org/jira/browse/METRON/?selectedTab=com.atlassian.jira.jira-projects-plugin:summary-panel).
- [ ] Does your PR title start with METRON-XXXX where XXXX is the JIRA number you are trying to resolve? Pay particular attention to the hyphen "-" character.
- [ ] Has your PR been rebased against the latest commit within the target branch (typically master)?

### For code changes:
- [ ] Have you included steps to reproduce the behavior or problem that is being changed or addressed?
- [ ] Have you included steps or a guide to how the change may be verified and tested manually?
- [ ] Have you ensured that the full suite of tests and checks have been executed in the root metron folder via:
  ```
  mvn -q clean integration-test install && dev-utilities/build-utils/verify_licenses.sh 
  ```

- [ ] Have you written or updated unit tests and or integration tests to verify your changes?
- [ ] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under [ASF 2.0](http://www.apache.org/legal/resolved.html#category-a)?
- [ ] Have you verified the basic functionality of the build by building and running locally with Vagrant full-dev environment or the equivalent?

### For documentation related changes:
- [ ] Have you ensured that format looks appropriate for the output in which it is rendered by building and verifying the site-book? If not then run the following commands and the verify changes via `site-book/target/site/index.html`:

  ```
  cd site-book
  mvn site
  ```

#### Note:
Please ensure that once the PR is submitted, you check travis-ci for build issues and submit an update to your PR as soon as possible.
It is also recommended that [travis-ci](https://travis-ci.org) is set up for your personal repository such that your branches are built there before submitting a pull request.

### CODE REVIEW

File: metron-interface/metron-alerts/src/app/pcap/service/pcap.service.ts [hunk 1]
```diff
@@ -39,6 +39,1480 @@ function pdml_json() {
       "capture_file": "./metron-platform/metron-api/src/test/resources/test-tcp-packet.pcap"
     },
     "packet": [
+      {
```
Comment (simonellistonball) [inline metron-interface/metron-alerts/src/app/pcap/service/pcap.service.ts:42]: Is this just to add a second line to the test data?

File: metron-interface/metron-alerts/src/app/pcap/pcap-panel/pcap-panel.component.html [hunk 2]
```diff
@@ -1,7 +1,6 @@
-<button (click)="test()">Test</button>
-<div class="panel">
-  <div classs="panel-header">
-    <app-pcap-filters (search)="onSearch($event)"></app-pcap-filters>
+<div class="panel my-4">
+  <div class="panel-header">
+    <app-pcap-filters (search)="test()"></app-pcap-filters>
```
Comment (simonellistonball) [inline metron-interface/metron-alerts/src/app/pcap/pcap-panel/pcap-panel.component.html:3]: any reason to code in the test here?

**Review by simonellistonball — COMMENTED** [review]
Thanks @merrimanr had a look at the code and couple of questions

File: metron-interface/metron-alerts/src/app/pcap/service/pcap.service.ts (hunk 1 above)
Comment (merrimanr) [reply metron-interface/metron-alerts/src/app/pcap/service/pcap.service.ts:42]: Yes I wanted to make sure the styling looked right for multiple rows.

File: metron-interface/metron-alerts/src/app/pcap/pcap-panel/pcap-panel.component.html (hunk 2 above)
Comment (merrimanr) [reply metron-interface/metron-alerts/src/app/pcap/pcap-panel/pcap-panel.component.html:3]: I am assuming we will switch it back to the real call at some point.  Just wanted to give you a preview of how the panel with look and behave.  Happy to change it back now or leave this PR open and add to it until we get the backend integration done.

File: metron-interface/metron-alerts/src/app/pcap/service/pcap.service.ts (hunk 1 above)
Comment (simonellistonball) [reply metron-interface/metron-alerts/src/app/pcap/service/pcap.service.ts]: fair enough. we will have to take out my dodgy hack test button anyway.

File: metron-interface/metron-alerts/src/app/pcap/pcap-panel/pcap-panel.component.html (hunk 2 above)
Comment (simonellistonball) [reply metron-interface/metron-alerts/src/app/pcap/pcap-panel/pcap-panel.component.html:3]: fair enough. we will have to take out my dodgy hack test button anyway.

**Pull request merged**